CoroutineTemplate/domain/src/main/java/co/nimblehq/coroutine/domain/usecase/UseCase.kt

```diff
@@ -0,0 +1,6 @@
+package co.nimblehq.coroutine.domain.usecase
+
+import co.nimblehq.coroutine.domain.repository.Repository
+import javax.inject.Inject
+
+class UseCase @Inject constructor(private val repository: Repository)
```

Comment:
Should we consider adding example code for the `operator` function like this discussion 🤔

Reply:
@thiennguyen0196 Could you please write an example of what you have in mind to add to this class? 🙏

Reply:
Sound neat to me @Tuubz 😆